resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -463,6 +466,20 @@ public DnsNameResolverBuilder decodeIdn(boolean decodeIdn) {
         return this;
     }
 
+    /**
+     * Set the maximum numbers of lookups that we try to consolidate when in-flights. This means if multiple
```

Comment:
Just a tiny thing. Thank you!
```suggestion
     * Sets the maximum numbers of lookups that we try to consolidate when in-flights. This means if multiple
```